src/drafts/Prose/Prose.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react'
```

Comment:
I'm questioning whether PRC even needs this component. It's very rare that we need to render a large block of prose in the apps that use Primer.

Reviewers - do you think we should add a Prose component?

Reply:
@mperrotti I'd be down to ship with the underline changes that you have with the custom property stuff and then see if we still need something like this in the future 👀 Like you said, it seems like underline behavior is more of a one-off thing.

Reply:
Thanks @joshblack - I totally missed your comment last week. I'll take out the Prose stuff.